pages/common/smartctl.md

```diff
@@ -1,7 +1,7 @@
 # smartctl
 
 > View a disk's SMART data and other information.
-> See https://en.wikipedia.org/wiki/S.M.A.R.T for more information.
+> See [en.wikipedia.org/wiki/S.M.A.R.T.](https://en.wikipedia.org/wiki/S.M.A.R.T.) for more information.
```

Comment:
Hi @mueller-ma, 

Thank you for the contribution 👍 
I believe we should not be using markdown because the text from this command description might be rendered in clients that do not support that. 
I think the direct link (without the square brackets and parenthesis) should be used in this case.

Reply:
Actually, most markdown is ok - the spec requires clients implement full CommonMark support.

It's just that the regular link syntax isn't well supported (despite the fact that there is now [a standard](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) for displaying it in a terminal).

Reply:
I personally like having "https://" hidden in weblinks. I suggest you add a sentence about links to your Style Guide.

Off topic: You can also add a sentence about mail addresses and urls in example must be `@example.com`, because this is exactly why example.com exists.

Reply:
Great ideas!

Reply:
I can do so, if you tell me how you want to have hyperlinks.

Reply:
I think we'd prefer the prefix to be shown in hyperlinks, and use the `<....>` syntax if possible.